### PR TITLE
adding spotlight for donations blurb

### PR DESCRIPTION
**Pull request by laurenelee**
## Description

Adding a brief blurb to the dev spotlight page announcing spotlight for donations
I've intentionally added it underneath the examples to the left column to differentiate it from the text-heavy portion of the right column.

### CODE REVIEW

File: config/landing_pages/spotlight.yml [hunk 1]
```diff
@@ -58,12 +58,23 @@ page:
           image: "https://www.nexmo.com/wp-content/uploads/2019/02/Build-a-One-Time-Password-OTP-Service-by-Using-the-Dispatch-API-768x384.png"
           url: "https://www.nexmo.com/blog/2019/02/07/build-an-otp-service-dr/"
           alt_text: "Build a One Time Password Service Using the Nexmo Dispatch API"
+      - type: structured_text
+        structured_text:
+          header: "Spotlight for Donations"
+          icon:
+            name: "icon-money"
```
Comment (mheap) [inline config/landing_pages/spotlight.yml]: This doesn't seem to be a valid icon name. I think `icon-refresh-dollar` is the closest we have

Comment (laurenelee) [reply config/landing_pages/spotlight.yml]: updated!

**Comment by laurenelee** [comment]
WIP - don't merge

**Comment by laurenelee** [comment]
Okay after a bunch of back and forth with Content Marketing team, we're finally ready for this to be merged team! Thank you!!

**Pull request merged**